src/Voron/Data/Tables/TableSchemaStatsReference.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Runtime.InteropServices;
```

Comment:
Using directive is not required by the code and can be safely removed

Reply:
done